web-app/src/app/screens/Feeds/index.tsx

```diff
@@ -194,351 +213,386 @@ export default function Feed(): React.ReactElement {
     setFeatureCheckboxData(generateCheckboxStructure());
   }, [selectedFeatures]);
 
+  const containerRef = React.useRef(null);
+  useEffect(() => {
+    const observer = new IntersectionObserver(
+      ([entry]) => {
+        setIsSticky(!entry.isIntersecting);
+      },
+      { threshold: 1.0 },
+    );
+
+    if (containerRef.current !== null) {
+      observer.observe(containerRef.current);
+    }
+
+    return () => {
+      observer.disconnect();
+    };
+  }, []);
+
   return (
-    <Container component='main' maxWidth='xl'>
+    <Container component='main' maxWidth={false}>
       <CssBaseline />
       <Box
         sx={{
           display: 'flex',
           flexDirection: 'column',
         }}
-        mx={{ xs: 0, m: 'auto' }}
+        mx={{ xs: 0, md: 'auto' }}
       >
-        <Grid container spacing={2}>
-          <Grid item xs={12}>
-            <Typography
-              component='h1'
-              variant='h4'
-              color='primary'
-              sx={{ fontWeight: 700 }}
-            >
-              {t('feeds')}
+        <Box sx={{ maxWidth: '1600px', mx: 'auto', width: '100%' }}>
```

Comment:
[question]: Are we planning at some point, we will have most of this style changes in a css/theme location?